.github/workflows/integration-tests-workflow.yml

```diff
@@ -150,10 +155,10 @@ jobs:
         if: ${{ runner.os == 'Linux' }}
         run: |
           cd ./OpenSearch-Dashboards
-          if timeout 600 grep -q "bundles compiled successfully after" <(tail -n0 -f dashboard.log); then
-            echo "OpenSearch Dashboards compiled successfully."
+          if timeout 60 grep -q "[OpenSearchDashboards][http] http server running at" <(tail -n0 -f dashboard.log); then
```

Comment:
we only need 1 min for time out here?

Reply:
Once compilation is done, startup is really fast (about 5 seconds from a cold start on my machine).

Reply:
I see, that make sense since we separated out the compilation step. LGTM!

Reply:
Ended up needing to refactor this to tail the whole file because the server was starting up _too_ fast, and so `tail` missed the "server running" line.